py4cast/datasets/base.py

```diff
@@ -533,6 +533,26 @@ def collate_fn(items: List[Item]) -> ItemBatch:
     return ItemBatch(**batch_of_items)
 
 
+def np_arange_include_end(start, end, step, dtype=None):
```

Comment:
nice doc ! wouldn't it work in any case with `np.arange(start, end + 1, step)` ?